### PR TITLE
Added Test Cases for Pod

### PR DESCRIPTION
**Pull request by rajdeepd**
Test case modified for Pod

### CODE REVIEW

**Comment by rajdeepd** [comment]
@thockin  Kindly review the pull request

**Comment by lavalamp** [comment]
ok to test

**Comment by rajdeepd** [comment]
@pwittrock @lavalamp Please approve when you get time

File: pkg/api/pod/util_test.go [hunk 1]
```diff
@@ -92,7 +92,88 @@ func TestFindPort(t *testing.T) {
...
+			name: "invalid ContainerPort",
+			containers: []api.Container{{}, {Ports: []api.ContainerPort{{
+				Name:          "a",
+				ContainerPort: -1,
+				Protocol:      "TCP",
+			},
+			}}},
+			port:     intstr.FromString("a"),
+			expected: -1,
+			pass:     true,
```
Comment (pwittrock) [inline pkg/api/pod/util_test.go]: Would you also add a comment that this should fail as well

Comment (rajdeepd) [inline pkg/api/pod/util_test.go]: done

**Pull request merged**